pkg/kv/kvserver/kvflowcontrol/rac2/token_counter.go

```diff
@@ -452,15 +464,31 @@ func WaitForEval(
 	m := len(scratch)
 	signaledQuorumCount := 0
 
+	var startWait time.Time
+	if traceIndividualDelays {
+		startWait = clock.PhysicalTime()
+	}
+	traceDelayWithReason := func(reason string) {
+		nextStartWait := clock.PhysicalTime()
+		waitDur := nextStartWait.Sub(startWait)
+		startWait = nextStartWait
+		log.Eventf(ctx, "wait-for-eval waited %s for %s", waitDur, reason)
```

Comment:
Should this be `log.VEventf`, like the other call above? Or vice versa, the call above be `log.Eventf`? Or we need them differ for some reason?

Reply:
It doesn't need VEventf because it only traces when `traceIndividualDelays = true`, which occurs when `vmodule=range_controller=2` or tracing is enabled on the ctx.